sound_play/include/sound_play/sound_play.h

```diff
@@ -62,58 +61,57 @@ namespace sound_play
 class SoundClient
 {
 public:
-	class Sound
+    class Sound
+    {
+	friend class SoundClient;
+    private:
+	int snd_;
+	float vol_;
+	std::string arg_;
+	std::string arg2_;
+	SoundClient *client_;
```

Comment:
if we are going to update this code styling, we should probably follow the ROS style guide: http://wiki.ros.org/CppStyleGuide, but this code uses tabs rather than 2 spaces per indent.